source/_integrations/upb.markdown

```diff
@@ -34,7 +38,7 @@ An event is generated whenever a UPB Link is:
 - fade is stopped
 - blink is started
 
-The event is `upb.link_changed`.
+The event is `upb.scene_changed`.
```

Comment:
Could you point me to the code?
As for as I can see, there is only 1 even fired: <https://github.com/home-assistant/core/blob/dev/homeassistant/components/upb/__init__.py#L52>

Constant definition:
<https://github.com/home-assistant/core/blob/dev/homeassistant/components/upb/const.py#L16>

Reply:
I was informed [here](https://community.home-assistant.io/t/upb-lighting/615/289?u=123) that the code would be changed and that I should create a PR to update the documentation.

(See gwww's comment below.)